apps/design/frontend/src/geography_screen.tsx

```diff
@@ -406,10 +414,10 @@ function PrecinctForm({
   precinctId?: PrecinctId;
   savedPrecincts: Precinct[];
   districts: readonly District[];
-}): JSX.Element {
-  const [precinct, setPrecinct] = useState<Precinct>(
+}): JSX.Element | null {
+  const [precinct, setPrecinct] = useState<Precinct | undefined>(
     precinctId
-      ? find(savedPrecincts, (p) => p.id === precinctId)
+      ? savedPrecincts.find((p) => p.id === precinctId)
       : // To make mocked IDs predictable in tests, we pass a function here
         // so it will only be called on intial render.
```

Comment:
If you make any other changes to this PR, maybe also correct the spelling of `initial`?

Reply:
Fixed, thanks